config/environments/uat.rb

```diff
@@ -74,6 +74,8 @@
         OMINRT
   ]
 
+  # config.ceos_agencies = []
+
```

Comment:
Does this mean none of these data centers exist in UAT?

Reply:
Didn't check the UAT or SIT results.  The comment is there in case there are other data centers different from the ones in application.rb, which I doubt.